tests/e2e/specs/Multiselect.spec.js

```diff
@@ -122,4 +122,45 @@ describe('Multiselect', () => {
       });
     });
   });
+
+  it('should not move multiple shapes when shift + combo was held down', () => {
+    [
+      {combo: 'meta', task1Position: { x: 100, y: 100 }},
+      {combo: 'ctrl', task1Position: { x: 400, y: 100 }},
+      {combo: 'alt', task1Position: { x: 700, y: 100 }},
+    ].forEach(({combo, task1Position}) => {
+      const task2Position = { x: task1Position.x + 100, y: task1Position.y + 100 };
+      const translateAmount = 200;
+      const newTask1Position = { x: task1Position.x + translateAmount, y: task1Position.y + translateAmount };
+
+      dragFromSourceToDest(nodeTypes.task, task1Position);
+      dragFromSourceToDest(nodeTypes.task, task2Position);
+
+      cy.get('.paper-container').click();
```

Comment:
I realize this was taken from the above test, and the same applies there as well, but since the `.paper-container` selector is used here first, I think it would make more sense to specify the alias here.

```suggestion
      cy.get('.paper-container').as('paperContainer').click();
```